src/main/java/com/kodilla/ecommercee/CartController.java

```diff
@@ -1,4 +1,39 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.domain.Cart;
+import com.kodilla.ecommercee.domain.CartDto;
+import com.kodilla.ecommercee.domain.Product;
+import com.kodilla.ecommercee.mapper.CartMapper;
+import com.kodilla.ecommercee.service.CartService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("v1/cart")
 public class CartController {
-}
+    @Autowired
+    CartService cartService;
+
+    @Autowired
+    CartMapper cartMapper;
+
+    @RequestMapping(method = RequestMethod.GET, value="createEmptyCart")
```

Comment:
Zerknij na temat pisanai mapowań w REST na:
https://restfulapi.net/resource-naming/
W szczególności na kawałek "Never use CRUD function names in URIs"
I spróbuj dostosować te mapowania do tego opisu.

Przy okazji jako ciekawostka te mapowania można w nowszych wersjach Springa pisać nieco krócej. 
Zobacz na: https://www.baeldung.com/spring-new-requestmapping-shortcuts
I spróbuj też pod to przerobić